electron/src/js/menu/system.ts

```diff
@@ -319,13 +323,13 @@ const win32Template = {
     signOutTemplate,
     {
       accelerator: 'Alt+F4',
-      click: async () => await lifecycle.quit(),
+      click: async () => lifecycle.quit(),
```

Comment:
Do we no longer have to await the call if labeled with async?

Reply:
:+1: I will remove the `async` tag since `lifecycle.quit()` doesn't return a Promise.